copier/errors.py

```diff
@@ -90,7 +89,9 @@ class CopierAnswersInterrupt(CopierError, KeyboardInterrupt):
 
     """
 
-    def __init__(self, answers: 'AnswersMap', last_question: 'Question', template: 'Template') -> None:
+    def __init__(
+        self, answers: "AnswersMap", last_question: "Question", template: "Template"
```

Comment:
Why this change if you're not updating black?

Reply:
🤔 this was a result of me running `pre-commit run -a`

Reply:
well i can drop all the commits except bumping the pre-commit hooks w/ `pre-commit autoupdate`

Reply:
☝🏽 🧐  How is pre-commit/black passing since those single quotes?

Reply:
I think this is an important point. Maybe the test is buggy?

Reply:
Indeed, fixed in https://github.com/copier-org/copier/pull/511